File: CHPT/source/trunk/client_app/Model/tr_user_role.cs
using System;
namespace Model
{
	/// <summary>
	/// 人员与角色关系
	/// </summary>
	[Serializable]
	public partial class tr_user_role
	{
		public tr_user_role()
		{}
		#region Model
		private string _user_role_id;
		private string _user_id;
		private string _role_id;
		/// <summary>
		/// 
		/// </summary>
		public string user_role_id
		{
			set{ _user_role_id=value;}
			get{return _user_role_id;}
		}
		/// <summary>
		/// 人员ID，关联人员表
		/// </summary>
		public string user_id
		{
			set{ _user_id=value;}
			get{return _user_id;}
		}
		/// <summary>
		/// 角色ID，关联角色表
		/// </summary>
		public string role_id
		{
			set{ _role_id=value;}
			get{return _role_id;}
		}
		#endregion Model

	}
}

File: CHPT/source/trunk/supp_app/SspDispatchService/src/com/ctfo/storage/model/permission/TrUserRole.java
package com.ctfo.storage.model.permission;

import java.io.Serializable;

/**
 * 
 * 
 * <p>
 * ----------------------------------------------------------------------------- <br>
 * 工程名 ： SspDispatchService <br>
 * 功能： 人员与角色关系<br>
 * 描述： 人员与角色关系<br>
 * 授权 : (C) Copyright (c) 2011 <br>
 * 公司 : 北京中交慧联信息科技有限公司 <br>
 * ----------------------------------------------------------------------------- <br>
 * 修改历史 <br>
 * <table width="432" border="1">
 * <tr>
 * <td>版本</td>
 * <td>时间</td>
 * <td>作者</td>
 * <td>改变</td>
 * </tr>
 * <tr>
 * <td>1.0</td>
 * <td>2014-11-4</td>
 * <td>xuehui</td>
 * <td>创建</td>
 * </tr>
 * </table>
 * <br>
 * <font color="#FF0000">注意: 本内容仅限于[北京中交慧联信息科技有限公司]内部使用，禁止转发</font> <br>
 * 
 * @version 1.0
 * 
 * @author xuehui
 * @since JDK1.6
 */
public class TrUserRole implements Serializable {

	/** */
	private static final long serialVersionUID = 7002929146858709572L;

	/** user_role_id */
	private String userRoleId;

	/** 人员ID，关联人员表 */
	private String userId;

	/** 角色ID，关联角色表 */
	private String roleId;

	/** 服务站id，云平台用 */
	private String serStationId;

	/** 帐套id，云平台用 */
	private String setBookId;

	public String getSerStationId() {
		return serStationId;
	}

	public void setSerStationId(String serStationId) {
		this.serStationId = serStationId;
	}

	public String getSetBookId() {
		return setBookId;
	}

	public void setSetBookId(String setBookId) {
		this.setBookId = setBookId;
	}

	public String getUserRoleId() {
		return userRoleId;
	}

	public void setUserRoleId(String userRoleId) {
		this.userRoleId = userRoleId;
	}

	public String getUserId() {
		return userId;
	}

	public void setUserId(String userId) {
		this.userId = userId;
	}

	public String getRoleId() {
		return roleId;
	}

	public void setRoleId(String roleId) {
		this.roleId = roleId;
	}
}